pkg/occlient/occlient.go

```diff
@@ -380,10 +385,25 @@ func getExposedPorts(image *imagev1.ImageStreamImage) ([]corev1.ContainerPort, e
 	return ports, nil
 }
 
+func GetDefaultBuilderImages() ([]string, error) {
+	return GetImageStreams(openShiftNameSpace)
+}
+
+func GetImageStreams(namespace string) ([]string, error) {
```

Comment:
It might be better to call this function `GetImageStreamNames` if it is returning just names.
Also comment would be nice ;-)